client-app/shared/catalog/components/product/variations.vue

```diff
@@ -1,21 +1,46 @@
 <template>
-  <ProductTitledBlock
+  <VcWidget
     v-if="!model.hidden && product.variations?.length"
+    class="variations"
+    size="lg"
     :title="model.title || $t('shared.catalog.product_details.variations_block_title')"
-    icon="adjustments"
+    prepend-icon="cube"
   >
-    <div class="space-y-5">
-      <ProductVariationCard :variation="product" />
+    <VcLineItems :with-header="false">
+      <template #line-items>
+        <VcLineItem
+          v-for="variation in [product, ...product.variations]"
+          :key="variation.code"
+          with-image
+          with-price
+          with-properties
+          :image-url="variation.images[0].url"
```

Comment:
images[0] could be undefined -> images[0]?.url

Reply:
done